include/nuttx/timers/arch_rtc.h

```diff
@@ -46,7 +46,7 @@ void up_rtc_set_lowerhalf(FAR struct rtc_lowerhalf_s *lower);
 
 #else
 
-#  define up_rtc_set_lowerhalf(lower)
+#define up_rtc_set_lowerhalf(lower)
```

Comment:
Please review all similar change and revert it.

Reply:
ok